content/posts/newsletter/Newsletter for 2020-01-10.md

```diff
@@ -0,0 +1,167 @@
+
+![Monal iOS](https://framapic.org/ZyopsYd66GsN/aOp2n1IrXfIy.png)
+
+Gajim has posted their [Development News](https://gajim.org/post/2020-01-07-development-news-december/) for December 2019: old message dialog system, compact tooltip when hovering a group chat in Gajim’s contact list, new assistant for password change and account removal, HTTP File Upload now work asynchronously with the ‘libsoup’ library, bug fixes, plugin updates, and changes in python-nbxmpp.
+
+Tigase has released [BeagleIM 3.4 and Siskin IM 5.4](https://tigase.net/beagleim-3.4-and-siskin-5.4-released/): BeagleIM 3.4 allows you to set MUC room avatar, simplifies MUC room settings, and bug fixes, while SiskinIM 5.4 adds support for setting MUC room avatar.
+
+[Movim 0.16.1](https://nl.movim.eu/?node/pubsub.movim.eu/Movim/cdfc0a4c-3459-4d3b-8c15-08994810d54e) has been released: you can now share posts to your connected chatrooms, communities layout were redesigned, all the messages that you sent in the one to one discussions can now be edited, the videoconferencing feature was heavily refactored, and the bug fixes.
+
+[Profanity 0.8.0](https://twitter.com/ProfanityIM/status/1207735919488438273) has been released with consistent color generated nicks (XEP-0392) and a trackbar so you see where you left off reading!
+
+Goffi has published the [SàT progress note 2019-W51](https://www.goffi.org/b/JLPTTHfRnqhxNozHjRz553/progress-note): infinite scroll is now implemented in Cagou, Kivy community extensions used by SàT are now installable like any other Python dependency, e2e encryption with OMEMO, and more.
+
+### Libraries
+
+[gloox 1.0.23 released](https://camaya.net/news/gloox-1-0-23-released/) as a maintenance version.
+
+## GSoC 
+
+Google Summer of Code 2020 has been announced, some XMPP projects have already shown interest to mentor student.
+
+## Extensions and specifications 
+
+### New 
+
+#### XEP-0426: Character counting in message bodies 
+
+Version 0.1.0 of XEP-0426 (Character counting in message bodies) has
+been released.
+
+Abstract:
+This document describes how to correctly count characters in message
+bodies. This is required when referencing a position in the body.
+
+Changelog:
+Promote to Experimental as per Council decision. (mb)
+
+URL: https://xmpp.org/extensions/xep-0426.html
+
+### Proposed 
+
+#### Character counting in message bodies 
```

Comment:
I'm not sure if we should mention the same XEP two times (in New and in Proposed)

Reply:
I had that doubt as well, but we should mention that it has been through two states: "Proposed" and then "New", right?